README.md

```diff
@@ -95,6 +95,26 @@ You should now see that for every Article in the Kontent site there's an html fi
 
 This is a very basic pipeline and gives you everything you need to get started with content served from a Kontent project. But Kontent has more advanced features and some nice extras that you can leverage to get better integration and more robust code.
 
+## Filtering content
+
+If you need to filter out the the input document s from Kontent, it is possible to specify the query of your request using `WithQuery(IQueryParameter[])` method.
+
+Let's say, you just want first three latest articles that have the Title element set and you want to load only a subset of.
+
+```csharp
+// ...
+new Kontent<Article>(client)
+    .WithQuery(
+        new NotEmptyFilter("elements.title"),
```

Comment:
The model generator provides all the strings used in this query. As a coding best practice I'd use those strings as much as possible

Reply:
Do you mean something like that?

```suggestion
        new NotEmptyFilter("$"elements.{Article.TitleCodename}""),
```

Reply:
Yes, that's what I meant. @petrsvihlik is there a more elegant way to handle the "elements" part?

Reply:
This is currently the best way to do it...I tried to outline a solution here: https://github.com/Kentico/kontent-delivery-sdk-net/issues/241#issuecomment-724751872
Please comment.